pilot/pkg/xds/monitoring.go

```diff
@@ -118,6 +118,12 @@ var (
 	ldsSendErrPushes = pushes.With(typeTag.Value("lds_senderr"))
 	rdsSendErrPushes = pushes.With(typeTag.Value("rds_senderr"))
 
+	pushContextInitTime = monitoring.NewDistribution(
+		"pilot_pushcontext_init_time",
```

Comment:
Based on https://prometheus.io/docs/practices/naming/#metric-names, I think this should be `pilot_pushcontext_init_seconds` perhaps?

cc @douglas-reid

Reply:
yes, let's use the units.

btw, @howardjohn are we going to use `pilot` forever?  or should new metrics get `istiod` prefixes?  wdyt?

Reply:
Its a good question that I don't have a good answer to. It seems nice to have things be consistent, and I could see the argument that its the "pilot component within Istiod"... if we did it again surely it would be `istiod_` (or maybe even `istio_`), but might not be worth changing

Reply:
We'd better not change istiod_xxx, those metrics are to end users, we should not change over and over.

Reply:
Yes, I agree it is too late to change existing metrics (at least, without a long transition). To me, the question is *new* metrics. I wonder if "pilot_" means anything to users at this point (particularly new users). I almost buy the component-within-istiod bit, but it feels like then it should be `istiod_pilot_....` (generally the single word prefix is the name of the application).

Reply:
I donot think it should be so complex, just a name consistent with previous ones.